### PR TITLE
Extract and use plugin images digest from env.

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
### What does this PR do?
https://github.com/eclipse/che/issues/16815

### What issues does this PR fix or reference?
Extract and use plugin images digest from env.

### Depends on:
https://github.com/eclipse/che-operator/pull/274

Signed-off-by: Oleksandr Andriienko <oandriie@redhat.com>

### CODE REVIEW

File: build/dockerfiles/entrypoint.sh [hunk 1]
```diff
@@ -37,6 +37,47 @@ METAS_DIR="${METAS_DIR:-${DEFAULT_METAS_DIR}}"
 #   \7 - Optional quotation following image reference
 IMAGE_REGEX='([[:space:]]*"?)([._:a-zA-Z0-9-]*)/([._a-zA-Z0-9-]*)/([._a-zA-Z0-9-]*)(@sha256)?:([._a-zA-Z0-9-]*)("?)'
 
+if [[ ! -z $(env | grep .*plugin_registry_image.*) ]];then
```
Comment (tolusha) [inline build/dockerfiles/entrypoint.sh:40]: it is worth adding some comments explaining what the following extract does.

**Comment by benoitf** [comment]
hi, should we use yq to hande yaml instead of grep/sed ?

**Comment by AndrienkoAleksandr** [comment]
> hi, should we use yq to hande yaml instead of grep/sed ?

We, can, but during work on pr https://github.com/eclipse/che-operator/pull/274 I found that yq is much slower, that sed...

**Comment by AndrienkoAleksandr** [comment]
> /bin/bash: .ci/cico-plugin-registry-pr-test.sh: No such file or directory

Ohrimenko1988 Sorry, do we have such test? In the repository I don't see it.

**Comment by amisevsk** [comment]
>> hi, should we use yq to hande yaml instead of grep/sed ?
> We, can, but during work on pr eclipse/che-operator#274 I found that yq is much slower, that sed...

I've found that most of the cost of `yq` is in invoking it, and if you can collapse as much as possible into a single script it's about as fast as a sed/grep solution. That may be infeasible for this task, however.

**Pull request merged**